app/components/select/state-select.js

```diff
@@ -0,0 +1,63 @@
+import Ember from 'ember';
+
+const {
+  Component
+} = Ember;
+
+export default Component.extend({
+  classNames: ['select-state'],
+  stateOptions: [
+    { name: 'Alabama', id: 'AL' },
```

Comment:
I would note this article link in here for documentation: https://support.stripe.com/questions/can-i-use-stripe-if-my-business-is-based-in-an-outlying-territory-of-a-supported